db/seeds.rb

```diff
@@ -6,7 +6,7 @@
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
 
-user1 = User.create!(username: 'Luma')
+user1 = User.create!(username: 'Luma', email: 'test@email.com', password: '12345678', password.confirmation: '12345678' )
```

Comment:
At the moment, the user1 seed is not properly being created. Let's change the `password.confirmation` to `password_confirmation`